tokio/src/runtime/scheduler/multi_thread/handle/metrics.rs

```diff
@@ -8,7 +8,8 @@ impl Handle {
     }
 
     pub(crate) fn num_blocking_threads(&self) -> usize {
-        self.blocking_spawner.num_threads()
+        // workers are currently spawned within the blocking_spawned
+        self.blocking_spawner.num_threads() - self.num_workers()
```

Comment:
It could underflow during runtime startup.
```suggestion
        self.blocking_spawner.num_threads().saturating_sub(self.num_workers())
```

Reply:
Ahh good point. I thought about this but opted for the underflow to cause a panic should the way workers are spawned change in the future, but good point that this could underflow at startup.

Reply:
Hopefully there are enough tests that they will catch it if we change how it works. :)